index.html

```diff
@@ -1,248 +1,292 @@
         </div>
+      </div>
     </section>
 
     <section id="portfolio-skills">
-        <div class="portfolio-details">
-            <img src="assets/languages-icon.svg" alt="">
-            <h4>languages</h4>
-
-            <ul class="portfolio-languages">
-                <li>Javascript</li>
-                <li>Ruby</li>
-                <li>HTML</li>
-                <li>CSS</li>
-            </ul>
-        </div>
+      <div class="portfolio-details">
+        <img src="assets/languages-icon.svg" alt="" />
+        <h4>languages</h4>
 
-        <div class="portfolio-details">
-            <img src="assets/icon-frameworks.svg" alt="">
-            <h4>Frameworks</h4>
+        <ul class="portfolio-languages">
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+          <li>CSS</li>
+        </ul>
+      </div>
 
-            <ul class="portfolio-languages">
-                <li>React.js</li>
-                <li>Ruby on Rails</li>
-                <li>RSpec</li>
-                <li>Capybara</li>
-                <li>Selenium</li>
-            </ul>
-        </div>
+      <div class="portfolio-details">
+        <img src="assets/icon-frameworks.svg" alt="" />
+        <h4>Frameworks</h4>
 
-        <div class="portfolio-details">
-            <img src="assets/skills-icon.svg" alt="">
-            <h4>Skills</h4>
+        <ul class="portfolio-languages">
+          <li>React.js</li>
+          <li>Ruby on Rails</li>
+          <li>RSpec</li>
+          <li>Capybara</li>
+          <li>Selenium</li>
+        </ul>
+      </div>
 
-            <ul class="portfolio-languages">
-                <li>Database Management</li>
-                <li>Version Control</li>
-                <li>Cli</li>
-                <li>Web Development</li>
-                <li>Api Design</li>
-            </ul>
-        </div>
+      <div class="portfolio-details">
+        <img src="assets/skills-icon.svg" alt="" />
+        <h4>Skills</h4>
 
-        
+        <ul class="portfolio-languages">
+          <li>Database Management</li>
+          <li>Version Control</li>
+          <li>Cli</li>
+          <li>Web Development</li>
+          <li>Api Design</li>
+        </ul>
+      </div>
     </section>
 
+    <section id="contact-form">
+        <p>I'm always interested in hearing about new projects, so if you you'd like to chat please get in touch</p>
+        <div class="inputs">
+            <form class="contact" action="https://formspree.io/f/mgebgadd" method="post">
+                <input class="form1" type="text" id="name" type="text" placeholder="Full name" maxlength="30"   required ><br>
+                <input class="form2" name="Email" id="email" type="email" placeholder="Email address"   required><br>
+                <textarea class="text_form" name="textarea" id="textarea" placeholder="Hey I'd like to connect to..." maxlength="500" rows="5"  required></textarea><br>
```

Comment:
- [ ] Kindly remove the `br` tag from your code, it seems like you used it intentionally to make the `input` should visible one after the other because `input` by default has the behaviour of `inline` **flex** display, but I would recommend you to delete it because as per the requirement, you should use the `flexbox` to align your form content `properly`, with the help of which you can easily achieve the same result but in a better way, especially it will help you while implementing the `desktop` **version** of your portfolio.

Reply:
yes i already remove the br tag